services/rfid/lib/rfid.js

```diff
@@ -0,0 +1,88 @@
+let mfrc522 = null;
+
+const DEBUG = process.env.DEBUG;
+
+// Try to handle not running on a Pi more gracefully
+try {
+  mfrc522 = require('mfrc522-rpi');
+} catch (e) {
+  console.warn('RFID library not found. Are you on a Pi?\n\n');
+
+  if (!DEBUG) {
+    throw new Error(e);
+  }
+}
+
+if (mfrc522 == null) {
+  mfrc522 = {
+    initWiringPi: () => {},
+    reset: () => {},
+    findCard: () => ({ status: '💯' }),
+    getUid: () => ({ status: '💯', data: new Buffer('FAKE') })
+  };
+}
+
+// Init WiringPi with SPI Channel 0
+mfrc522.initWiringPi(0);
```

Comment:
I think that anything that isn't purely setup should be executed in the exported function.